modules/serialize/test/ecdsa_signatures.test.ts

```diff
@@ -1620,24 +458,57 @@ describe('raw2der', () => {
     )
   })
 
-  it('transform to der signature with no padding', () => {
-    expect(raw2der(rawSigNoPadding, suite)).to.deep.equal(derSigNoPadding)
-  })
-  it('transform to der signature with r padded', () => {
-    expect(raw2der(rawSigRPadded, suite)).to.deep.equal(derSigRPadded)
-  })
-  it('transform to der signature with s padded', () => {
-    expect(raw2der(rawSigSPadded, suite)).to.deep.equal(derSigSPadded)
-  })
-  it('transform to der signature with both r and s padded', () => {
-    expect(raw2der(rawSigBothSandRPadded, suite)).to.deep.equal(
-      derSigBothSandRPadded
+  describe('padding', () => {
+    const suite = new WebCryptoAlgorithmSuite(
+      AlgorithmSuiteIdentifier.ALG_AES256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384
     )
-  })
-  it('transform to der signature with with r padded, but r is on the padding boundary', () => {
-    expect(raw2der(rawSigRonBoundary, suite)).to.deep.equal(derSigRonBoundary)
-  })
-  it('transform to der signature with s padded, but s is on the padding boundary', () => {
-    expect(raw2der(rawSigSonBoundary, suite)).to.deep.equal(derSigSonBoundary)
+
+    const commitSuite = new WebCryptoAlgorithmSuite(
+      AlgorithmSuiteIdentifier.ALG_AES256_GCM_IV12_TAG16_HKDF_SHA512_COMMIT_KEY_ECDSA_P384
+    )
+
+    it('No Padding', () => {
+      expect(raw2der(rawSigNoPadding, suite)).to.deep.equal(derSigNoPadding)
+    })
+
+    it('Original vector R padded', () => {
+      expect(raw2der(rawSigRPadded, suite)).to.deep.equal(derSigRPadded)
+    })
+
+    it('Original vector S padded', () => {
+      expect(raw2der(rawSigSPadded, suite)).to.deep.equal(derSigSPadded)
+    })
+
+    it('Original vector R and S padded', () => {
+      expect(raw2der(rawSigBothSandRPadded, suite)).to.deep.equal(
+        derSigBothSandRPadded
+      )
+    })
+
+    it('New vector S only padded', () => {
```

Comment:
Very minor: I hate/despise *New* in a repo. It is New today, it is old tomorrow. Is there some other adjective we can use that possible addresses the **Only** comment above?